pkg/ui/src/redux/apiReducers.ts

```diff
@@ -162,7 +161,7 @@ const problemRangesReducerObj = new KeyedCachedDataReducer(
 export const refreshProblemRanges = problemRangesReducerObj.refresh;
 
 export const certificatesRequestKey = (req: api.CertificatesRequestMessage): string =>
-  _.isEmpty(req.node_id) ? "" : req.node_id.toString();
+  _.isEmpty(req.node_id) ? "0" : req.node_id;
```

Comment:
do you need to change this and the other `"0"` to `"all"` as well?

Reply:
Doing that in a follow up PR already.  It might upset too much if I do it here.

Reply:
So, the page won't work until you merge the follow up PR? What else is in the follow up PR?

Reply:
Nah, this will always work.  The 0 doesn't matter.  There is no 0 node and we never request all certs at once, only one at a time.

Reply:
oh, didn't realize that those changes weren't related. Just prettier reformatting things? got me again.